ibis/tests/expr/test_temporal.py

```diff
@@ -705,3 +705,9 @@ def test_time_truncate(table, operand, unit):
     expr = operand(table).truncate(unit)
     assert isinstance(expr, ir.TimeValue)
     assert isinstance(expr.op(), ops.TimeTruncate)
+
+
+def test_date_time_literals():
+    ibis.date(2022, 2, 4)
+    ibis.time(16, 20, 00)
+    ibis.timestamp(2022, 2, 4, 16, 20, 00)
```

Comment:
```suggestion
    assert ibis.timestamp(2022, 2, 4, 16, 20, 00).type() == dt.timestamp
```

I think the important thing here is to test that the data type of the resulting expression is inferred correctly.

You may need to add `import ibis.expr.datatypes as dt`.

You may also need to run `black` and `isort` again after these changes.